src/main/java/console/output/ConsoleColors.java

```diff
@@ -0,0 +1,18 @@
+package console.output;
+
+public class ConsoleColors {
+
+    // Reset
+    public static final String RESET = "\033[0m";
+
+    // Regular Colors
+    public static final String BLACK = "\033[0;30m";
+    public static final String RED = "\033[0;31m";
+    public static final String GREEN = "\033[0;32m";
+    public static final String YELLOW = "\033[0;33m";
+    public static final String BLUE = "\033[0;34m";
+    public static final String PURPLE = "\033[0;35m";
+    public static final String CYAN = "\033[0;36m";
+    public static final String WHITE = "\033[0;37m";
+
+}
```

Comment:
Fantastic idea to move colors (consts) to another class!
Consider changing this class to an enum